### DISCUSSION TITLE
reload code without restart

### DISCUSSION THREAD

**Discussion by ThanatosDi** [Pycord-Development/pycord#1177] [Q&A]
hello everyone, how can I reload code without restart discord bot?
`I try bot.reload_extension('cog_name')`,  `bot.register_commands(bot.application_commands)` and `bot.sync_commands(bot.application_commands)`
I check discord debug log, it put commands return 200
but command return value still old code not new code.

**Comment by Dorukyum** [comment]
Does this persist with pycord installed from the master branch?
